src/components/game/LoadProgramError.tsx

```diff
@@ -45,7 +52,18 @@ function NoProgramError() {
   return <ErrorScreen>
     <p style={{ marginTop: '5px', marginBottom: '5px' }}>
       No se encontró un programa dentro del proyecto.
-      Podes crear uno con la extensión .wpgm dentro de la carpeta src para poder correr el juego.
+      Podés crear uno con la extensión .wpgm dentro de la carpeta src para poder correr el juego.
+    </p>
+    <div style={{ paddingTop: '2%' }}>
+      <BackArrow returnPath='/' />
+    </div>
+  </ErrorScreen>
+}
+
+function GenericError() {
+  return <ErrorScreen>
+    <p style={{ marginTop: '5px', marginBottom: '5px' }}>
+      Lo sentimos, ocurrió un error desconocido y no se pudo cargar el juego. Podés volver atrás e intentar cargar otro.
```

Comment:
```suggestion
      Lo sentimos, ocurrió un error y no se pudo cargar el juego.
```

Quedamos en agregar un cuadro scrolleable y mostrar el `error.message`.

Reply:
Me quedó así:

![image](https://user-images.githubusercontent.com/31800576/134279279-79db4645-64cf-4b97-bb0d-985d851dff8a.png)

Si tienen sugerencias para mejorarlo son bienvenidas jajaja